react/controls/FullscreenButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FunctionComponent } from 'react'
+import { IconExpand as DefaultIconFullscreen } from 'vtex.store-icons'
+
+import styles from '../styles/styles.css'
+
+export interface FullscreenButtonProps {
+  toggleFullscreenMode: () => void
+  cssHandles: any
+  IconFullscreen?: any
```

Comment:
I guess the type here would be `ReactNode`... not sure though, it could also be `ReactElement` 🤔 .

You can try them out in VSCode and choose the one that works best :)